packages/service-library/src/servicelib/redis/__init__.py

```diff
@@ -8,6 +8,11 @@
     ProjectLockError,
 )
 from ._models import RedisManagerDBConfig
+from ._project_document_version import (
+    PROJECT_DB_UPDATE_REDIS_LOCK_KEY,
+    PROJECT_DOCUMENT_VERSION_KEY,
+    increment_and_return_project_document_version,
```

Comment:
minor: do you really need to expose the keys? I would expect to have functions that make use of it, so that this does not leak